src/content/headsup/NonIdealHeadsup.tsx

```diff
@@ -173,48 +179,44 @@ export class ErrorHeadsUp extends React.PureComponent<
     const hasError = Object.getOwnPropertyNames(this.props.error).length > 0;
 
     return (
-      <div
-        className={classnames(
-          "r2c-repo-headsup",
-          "nonideal-headsup",
-          "error-headsup"
-        )}
-      >
-        <div className="error-briefing">
-          <div className="error-briefing-message">
-            <Icon
-              icon={IconNames.WARNING_SIGN}
-              className="error-icon"
-              intent={Intent.DANGER}
-            />
-            <div className="error-message-text">
-              Couldn't load Preflight. Check that <code>api.secarta.io</code> is
-              whitelisted in your browser.
-            </div>
-          </div>
-          {hasError && (
-            <div className="error-briefing-action">
-              <Button
-                onClick={this.handleToggleShowDetails}
-                className="error-message-show-more"
-                small={true}
-                minimal={true}
-              >
-                Show {this.state.showDetails ? "less" : "details"}
-              </Button>
-            </div>
-          )}
-        </div>
+      <>
+        <SimpleHeadsup
+          status="error"
+          icon={<MissingIcon />}
+          headline={`
+    Couldn't load Preflight. Check that ${<code>api.secarta.io</code>} is
+              whitelisted in your browser.`}
+          rightSide={this.renderRight(hasError)}
+        />
         {this.state.showDetails && (
           <div className="error-details">
             <pre className="error-code">{this.props.projectState}</pre>
             <pre className="error-raw">{JSON.stringify(this.props.error)}</pre>
           </div>
         )}
-      </div>
+      </>
     );
   }
 
+  private renderRight(hasError: boolean) {
+    if (hasError) {
+      return (
+        <div className="error-briefing-action">
+          <Button
+            onClick={this.handleToggleShowDetails}
+            className="error-message-show-more"
+            small={true}
+            minimal={true}
+          >
+            Show {this.state.showDetails ? "less" : "details"}
+          </Button>
+        </div>
+      );
+    } else {
+      return <></>;
```

Comment:
should probably return `null` here
```suggestion
      return null;
```